README.md

```diff
@@ -14,6 +14,23 @@ We catalog [select](CONTRIBUTING.md#adding-a-license) open source licenses with
 
 ## Run It On Your Machine
 
+### Managing Dependencies
+
+It may be the case that your system doesn't have the required dependencies. You will need 'cmake' and 'make' installed on your computer. 
+
+* For MacOS, use homebrew to update your dependencies (Install Homebrew from [here](https://brew.sh/)):
+```bash
+brew install make cmake
+```
+* For Linux/Ubuntu, use the apt-get tool to install the dependencies:
+```bash
+sudo apt-get install make cmake
+```
+
+### Installing and Running the tool
+
+Note that the third command (./script/bootstrap) may require you to use 'sudo' access for installation. Add 'sudo' right before the third command, if need be, but don't use root access.
```

Comment:
When would sudo be required to run the bootstrap script? It seems like if someone is changing file permissions or setting GEM_HOME, they know what they are doing.

Reply:
Oh I see that you wrote:

>  I had to figure out the dependencies and the 'sudo' access request myself in my MacOS system

Curious why that was, but I'll take your word for it. 😄

Reply:
I also had a similar issue like the one on #693. Regarding the sudo access, it's needed for the gems installation on Mac if you're not an admin user because gems get installed in /Library/ where sudo permissions are needed.